File: Personagens/Mutante/mutante.c
#include "mutante.h"

void iniciarMutante(Mutante *mutante) {

    mutante->vidaMaxima = 100;
    mutante->vida = mutante->vidaMaxima;

    mutante->forca = 0;
    mutante->resistencia = 0;
    mutante->evasao = 10;

    mutante->sangramento = 0;
    mutante->envenenamento = 0;
}

File: personagens/Mutante.java
package personagens;

public class Mutante {
    public int vida;
    public int vidaMaxima;
    public int forca;
    public int resistencia;
    public int evasao;

    public int sangramento;
    public int turnosSangramento;
    public int envenenamento;

    public Mutante() {
        vidaMaxima = 85;
        vida = vidaMaxima;
        forca = 7;
        resistencia = 0;
        evasao = 8;

        sangramento = 0;
        turnosSangramento = 0;
        envenenamento = 0;
    }
}
